integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/schema/management/ElasticsearchIndexSchemaManagerValidationOperation.java

```diff
@@ -51,7 +54,8 @@ public final CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
 				} ) );
 	}
 
-	protected abstract CompletableFuture<?> apply(IndexSchemaManager schemaManager, ContextualFailureCollector failureCollector);
+	protected abstract CompletableFuture<?> apply(IndexSchemaManager schemaManager, ContextualFailureCollector failureCollector,
+			OperationSubmitter operationSubmitter);
```

Comment:
Same here, no need to add a parameter if you're only ever be testing the blocking submitter.